app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AttachmentsController < ApplicationController
+
+  # DELETE /attachments/1
+  def destroy
+    Attachment.all[params[:id].to_i].delete
```

Comment:
oh, I missed parences, yeah it will work but will do not what you need - it will get All records, put them into array and extract from array element with specified number - it's bad approach, because it will take too much memory to allocate all records... You just have to get only one. You can do next for it: `Attachment.find(params[:id]).delete` or you can do what you proposed.